docs/_milestone_.md

```diff
@@ -0,0 +1,85 @@
+# December Milestone
+
+## Overview
+
+- Complete the epics:
+  - `Add Support for Partitioned Database`
+  - `Inclusion of related models`
+  - `Production deployment`
+
+## Goals
+
+- [ ] Inclusion of related models [MVP]
+      https://github.com/strongloop/loopback-next/issues/1352
+
+  - [ ] [5]Reject create/update requests when data contains navigational
+        properties https://github.com/strongloop/loopback-next/issues/3439
+
+- [ ] Authentication & Authorization
+      https://github.com/strongloop/loopback-next/issues/3902
+
+  - [ ] [5]Contribute OpenAPI spec pieces from extensions
+        https://github.com/strongloop/loopback-next/issues/3854
+  - [ ] [3]Add user profile factory for authentication modules
```

Comment:
I would say https://github.com/strongloop/loopback-next/issues/3846 is not a high priority.
"support token based authentication in API Explorer in shopping example" from the epic story is much more important, I don't remember if we have a story for it(probably not), I remember Raymond wants to have a default auth dialog for token based authentication, and I agree with him.

Reply:
I thought #3854 would allow that. no?

Reply:
@dhmlau #3854 is "allow extension contributes spec", "support token based authentication in API Explorer in shopping example" means our framework provide built-in spec

Reply:
yes